modules/hal_nordic/nrfx/nrfx_config_reserved_resources_ncs.h

```diff
@@ -40,32 +53,40 @@
  */
 #if defined(CONFIG_BT_LL_SW_SPLIT)
 #include <bt_ctlr_used_resources.h>
-#if defined(DPPI_PRESENT)
-#if defined(NRF53_SERIES)
+#if defined(CONFIG_SOC_SERIES_NRF51X) || defined(CONFIG_SOC_COMPATIBLE_NRF52X)
+#define NRFX_PPI_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
+#define NRFX_PPI_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
+#elif defined(CONFIG_SOC_COMPATIBLE_NRF53X)
 #define NRFX_DPPI0_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
 #define NRFX_DPPI0_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
-#elif defined(NRF54L_SERIES)
+#elif defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
 #define NRFX_DPPI10_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
 #define NRFX_DPPI10_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
 #endif
-#else /* defined(DPPI_PRESENT) */
-#define NRFX_PPI_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
-#define NRFX_PPI_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
-#endif /* defined(DPPI_PRESENT) */
 #endif /* defined(CONFIG_BT_LL_SW_SPLIT) */
 
 #if defined(CONFIG_BT_LL_SOFTDEVICE)
+/* Define auxiliary symbols needed for SDC device dispatch. */
+#if defined(CONFIG_SOC_COMPATIBLE_NRF52X)
+#define NRF52_SERIES
+#elif defined(CONFIG_SOC_COMPATIBLE_NRF53X)
+#define NRF53_SERIES
+#elif defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
+#define NRF54L_SERIES
+#elif defined(CONFIG_SOC_SERIES_NRF54HX)
+#define NRF54H_SERIES
+#endif
```

Comment:
Well, this is quite hacky and I suppose it may be a source of confusion in the future. But let's go with this right now (I don't see an easy applicable alternative) and solve this problem in a better way in follow-up PRs.